finality-aleph/src/abft/current.rs

```diff
@@ -81,9 +76,10 @@ pub fn create_aleph_config(
     session_id: SessionId,
     unit_creation_delay: UnitCreationDelay,
 ) -> Config {
-    let mut config = default_config(n_members.into(), node_id.into(), session_id.0 as u64);
-    config.delay_config.unit_creation_delay = unit_creation_delay_fn(unit_creation_delay);
-    config.max_round = MAX_ROUNDS;
-
-    config
+    let mut delay_config = default_delay_config();
+    delay_config.unit_creation_delay = unit_creation_delay_fn(unit_creation_delay);
+    match create_config(n_members.into(), node_id.into(), session_id.0 as u64, MAX_ROUNDS, delay_config, Duration::from_millis(SESSION_LEN_LOWER_BOUND_MS as u64)) {
+        Ok(config) => config,
+        Err(_) => panic!("Incorrect setting of delays. Make sure the total AlephBFT session time is at least {} ms.", SESSION_LEN_LOWER_BOUND_MS),
```

Comment:
Shouldnt the error have this msg?

Reply:
Unfortunately error doesn't contain any message. Tho, `AlephBFT` will generate a separate `error!` log with some relevant message. I'd leave it as it is, as this panic contains a message more relevant to `aleph-node` user - a suggestion about how exactly to fix their config.

Reply:
imho its responsibility that belongs to abft